demos/python_demos/object_detection_demo_yolov3_async/object_detection_demo_yolov3_async.py

```diff
@@ -146,14 +153,18 @@ def parse_yolo_region(blob, resized_image_shape, original_im_shape, params, thre
             # Depends on topology we need to normalize sizes by feature maps (up to YOLOv3) or by input shape (YOLOv3)
             w = w_exp * params.anchors[2 * n] / (resized_image_w if params.isYoloV3 else params.side)
             h = h_exp * params.anchors[2 * n + 1] / (resized_image_h if params.isYoloV3 else params.side)
+            classes = []
             for j in range(params.classes):
                 class_index = entry_index(params.side, params.coords, params.classes, n * side_square + i,
                                           params.coords + 1 + j)
-                confidence = scale * predictions[class_index]
-                if confidence < threshold:
-                    continue
-                objects.append(scale_bbox(x=x, y=y, h=h, w=w, class_id=j, confidence=confidence,
-                                          h_scale=orig_im_h, w_scale=orig_im_w))
+                classes.append(predictions[class_index])
+
+            label = argmax(classes)
```

Comment:
Why are you selecting the _greatest_ class index? Is there some significance to it?

Reply:
I'm not sure what do yo mean. But as you know, for each bbox model predicts objectness score and class probabilities. So, the real score is derived as multiplication of objectness score and highest class probability. Another way, you need only class with highest probability.

Reply:
Right, I misread the code. Maybe you could rename `classes` to `class_probabilities` to make it clearer?